httpd/handler.go

```diff
@@ -129,8 +131,18 @@ func (h Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		}
 		rp.ServeHTTP(w, r)
 		return
+	} else if mount.BaseDir != "" {
+		f, err := os.Open(mount.BaseDir + r.URL.Path)
```

Comment:
The user input needs to be sanitized here to make sure that nothing above `localDir` can be served.
Maybe `filepath.Clean()` and `strings.HasPrefix()` ?

Reply:
I've ended up using Join, Join also calls clean.